plugins/hls-explicit-record-fields-plugin/src/Ide/Plugin/ExplicitFields.hs

```diff
@@ -169,6 +176,11 @@ codeActionProvider ideState _ (CodeActionParams _ _ docId range _) = do
       , _data_ = Just $ toJSON uid
       }
 
+    isConvertible :: RecordInfo -> Bool
+    isConvertible = \case
+      RecordInfoApp _ (RecordAppExpr Unsaturated _ _) -> False
+      _ -> True
```

Comment:
Since it is only two cases, maybe we should be exhaustive?

Reply:
Maybe I misunderstood you, but I can't see how it is only two cases. This is what I come up with if I try to make this fully exhaustive:

```haskell
    isConvertible :: RecordInfo -> Bool
    isConvertible = \case
      RecordInfoApp _ (RecordAppExpr Unsaturated _ _) -> False
      RecordInfoApp _ (RecordAppExpr Saturated _ _) -> True
      RecordInfoPat {} -> True
      RecordInfoCon {} -> True
```

I personally find this a bit too verbose, but if that's the style the codebase favors, I would be happy to change.

Reply:
It is just my preference, but since you are maintaining the plugin right now, your opinion matters more than mine in style preferences. 

Also, thought it would be two, not 4, so yeah, that's more verbose than anticipated.